### PR TITLE
What's New: Add info about new behavior when you change the question type

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: docs/Creator-V2-Whats-New.md [hunk 1]
```diff
@@ -98,13 +98,19 @@ _Page navigator_
...
 
 ![Action buttons](images/creator-v2-action-buttons.png)  
 _Action buttons_
 
+If you click the button that changes the question type, you will see a drop-down list of all supported types. In Survey Creator V1, this list was limited to the types that were interchangeable with the current element type. With the full list, you can now add multiple default questions to the survey to specify their types afterwards.
+
+![List of supported question types](images/creator-v2-question-types.png)   
+_Drop-down list of supported question types_
 
+If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration.
```
Comment (andrewtelnov) [inline docs/Creator-V2-Whats-New.md:113]: The following line of code comeback to the old behavior in V1.
`SurveyCreator.settings.questionConvertMode = 1;`
In this case text can be converted into "comment" only, dropdown into checkbox, radiogroup and so on. The behavior we have in V1.

Comment (RomanTsukanov) [reply docs/Creator-V2-Whats-New.md]: This information is added.

**Review by andrewtelnov — APPROVED** [review]

File: docs/Creator-V2-Whats-New.md [hunk 2]
```diff
@@ -98,13 +98,23 @@ _Page navigator_
 
 <a id="action-buttons"></a> 
 
-### Action buttons arranged
-Creator actions which are available in different tabs (Designer, Logic, etc.) and elements (such as pages, panels, or questions) are now displayed into one line. You have programmatic access to actions - they are in your full control.
+### Action buttons rearranged
+
+Previously, action buttons were split between different tabs (Designer, Logic, etc.). Now, they are displayed within the question that you configure. You also can access the actions programmatically.
```
Comment (arminal) [inline docs/Creator-V2-Whats-New.md:103]:
```suggestion
Action buttons used to be split between different tabs (Designer, Logic, etc.). Now, these actions are displayed within the question that you configure and can also be accessed programmatically.
```

File: docs/Creator-V2-Whats-New.md [hunk 3]
```diff
@@ -98,13 +98,23 @@ _Page navigator_
...
 
 ![Action buttons](images/creator-v2-action-buttons.png)  
 _Action buttons_
 
+If you click the button that changes the question type, you will see a drop-down list of all supported types. In Survey Creator V1, this list was limited to the types that were interchangeable with the current element type. With the full list, you can now add multiple default questions to the survey to specify their types afterwards.
+
+![List of supported question types](images/creator-v2-question-types.png)   
+_Drop-down list of supported question types_
+
+If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration. If you want to revert to the old behavior (limit the list to interchangeable types only), use the following code:
```
Comment (arminal) [inline docs/Creator-V2-Whats-New.md:113]:
```suggestion
If you switch between non-interchangeable question types, the question configuration may be lost. Use Undo and Redo buttons to roll back or reinstate the previous configuration. To revert to old behavior (limit the list to interchangeable types only), use the following code:
```

**Comment by arminal** [comment]
#GrammarReview

Short document. I rephrased a few of your sentences to make them more concise.

**Pull request merged**